webapp/channels/src/components/claim/components/email_to_oauth.tsx

```diff
@@ -119,12 +119,12 @@ const EmailToOAuth = (props: Props) => {
                     />
                 </p>
                 <div className={classNames('form-group', {'has-error': serverError})}>
-                    <LocalizedInput
+                    <input
                         type='password'
                         className='form-control'
                         name='password'
                         ref={passwordInput}
-                        placeholder={placeholderPasswordMessage}
+                        placeholder={formatMessage({id: t('claim.email_to_oauth.pwd'), defaultMessage: 'Password'})}
```

Comment:
The `t` function should not be needed here anymore.

Reply:
Removed